mapactionpy_arcmap/MapChef.py

```diff
@@ -112,17 +131,19 @@ def cook(self, productName, countryName):
         self.mxd.save()
 
     def addRasterToLayer(self, dataFrame, rasterFile, layer, raster, display):
-        for lyr in arcpy.mapping.ListLayers(layer): 
+        for lyr in arcpy.mapping.ListLayers(layer):
             lyr.replaceDataSource(rasterFile, "FILEGDB_WORKSPACE", raster)
-            lyr.visible = False 
-            arcpy.mapping.AddLayer(dataFrame, lyr, "BOTTOM")            
+            lyr.visible = False
+            arcpy.mapping.AddLayer(dataFrame, lyr, "BOTTOM")
 
+    # APS 06/09/2019 Would it be appropriate to name this method `addVectorToLayer` or correspond with
+    # `addRasterToLayer`?
```

Comment:
Would it be appropriate to name this method `addVectorToLayer` or correspond with `addRasterToLayer`?